# HG changeset patch
# User Rik <rik@octave.org>
# Date 1501171505 25200
#      Thu Jul 27 09:05:05 2017 -0700
# Node ID 9648436433774b828650738d14d04fe86cbcd480
# Parent  7713107371374da1f4d99abe385bf8a494f910ee
settings-dialog.cc: Don't mix enum with bool which provokes compiler warning.

* settings-dialog.cc (read_lexer_settings): Use just a bool for input argument
to Qt setChecked function.

diff --git a/libgui/src/settings-dialog.cc b/libgui/src/settings-dialog.cc
--- a/libgui/src/settings-dialog.cc
+++ b/libgui/src/settings-dialog.cc
@@ -135,21 +135,21 @@ read_lexer_settings (Ui::settings_dialog
           else
             bg_color[i] = new color_picker (lexer->paper (styles[i]));
           bg_color[i]->setToolTip
             (QObject::tr ("Background color, pink (255,0,255) means default"));
         }
       attrib_font[0+3*i] = new QCheckBox (QObject::tr ("b", "short form for bold"));
       attrib_font[1+3*i] = new QCheckBox (QObject::tr ("i", "short form for italic"));
       attrib_font[2+3*i] = new QCheckBox (QObject::tr ("u", "short form for underlined"));
-      attrib_font[0+3*i]->setChecked (Qt::Checked && actual_font.bold ());
+      attrib_font[0+3*i]->setChecked (actual_font.bold ());
       attrib_font[0+3*i]->setObjectName (actual_name+"_bold");
-      attrib_font[1+3*i]->setChecked (Qt::Checked && actual_font.italic ());
+      attrib_font[1+3*i]->setChecked (actual_font.italic ());
       attrib_font[1+3*i]->setObjectName (actual_name+"_italic");
-      attrib_font[2+3*i]->setChecked (Qt::Checked && actual_font.underline ());
+      attrib_font[2+3*i]->setChecked (actual_font.underline ());
       attrib_font[2+3*i]->setObjectName (actual_name+"_underline");
       color[i] = new color_picker (lexer->color (styles[i]));
       color[i]->setObjectName (actual_name+"_color");
       bg_color[i]->setObjectName (actual_name+"_bg_color");
       int column = 1;
       style_grid->addWidget (description[i],     i, column++);
       style_grid->addWidget (select_font[i],     i, column++);
       style_grid->addWidget (font_size[i],       i, column++);
